lib/action_cable/connection/callbacks.rb

```diff
@@ -0,0 +1,61 @@
+module ActionCable
+  module Connection
+    # = Action Cable Callbacks
+    #
+    # Action Cable provides hooks during the life cycle of a connection.
+    # Callbacks allow you to trigger logic during the life cycle of a
```

Comment:
Lets rewrite this sentence and the one before it to one sentence - they mostly repeat each other.